src/config-local.tsx

```diff
@@ -43,6 +43,15 @@ export default {
 
   BASIC_CHANNEL_ID: 0,
   INCENTIVIZED_CHANNEL_ID: 1,
-  ACTIVE_CHANNEL: Channel.INCENTIVIZED
-
+  ACTIVE_CHANNEL: Channel.INCENTIVIZED,
+
+    PARACHAIN_LIST: [
+      { "parachainName":'Snowbridge', "parachainId": 0, 'isDisabled':false },
+      { "parachainName":'Snowbridge-test', "parachainId": 1001,'isDisabled':false },
+      { "parachainName":'Acala (coming soon...)', "parachainId": 1010,'isDisabled':true },
```

Comment:
We should set the parachainId to 0 for disabled parachains. We do not know the ids yet.